osCam/streamin/views.py

```diff
@@ -1,9 +1,31 @@
-import imp
+# import imp
 from django.shortcuts import render
 from django.shortcuts import HttpResponse
 from django.http import JsonResponse
 # Create your views here.
+from django.shortcuts import render
+# from .forms import UploadFileForm
+from django.views.decorators.csrf import ensure_csrf_cookie
+
+# @ensure_csrf_cookie
+# def upload_display_video(request):
+#     if request.method == 'POST':
+#         form = UploadFileForm(request.POST, request.FILES)
+#         if form.is_valid():
+#             file = request.FILES['file']
+#             #print(file.name)
+#             handle_uploaded_file(file)
+#             return render(request, "upload-display-video.html", {'filename': file.name})
+#     else:
+#         form = UploadFileForm()
+#     return render(request, 'upload-display-video.html', {'form': form})
```

Comment:
is any of this commented out code needed?

Reply:
looks like only a few extra imports were added, but nothing else except for comments?